DiceKeys/Views/Screens/DiceKeyPresent.swift

```diff
@@ -222,6 +222,7 @@ struct DiceKeyPresent: View {
                     switch self.pageContent {
                     case .Save: DiceKeyStorageOptions(diceKey: diceKey, done: { navigate(to: .Default) }).padding(.horizontal, defaultContentPadding)
                     case .Derive(let derivationRecipeBuilder): DiceKeyWithDerivedValue(diceKey: diceKey, derivationRecipeBuilder: derivationRecipeBuilder)
+                        .frame(height: geometry.size.height * 0.7)
```

Comment:
If limiting a height, presumably it would be with `maxHeight` rather than `height`.

0.7 seems very brittle and dependent on screen dimensions.  Is there no better way to say that the top should be allowed shrink and the bottom should be guaranteed not to shrink?

Reply:
Yes, 0.3 will remain for bottom bar. Because there in Spacer() DiceKeyWithDerivedValue in this View. Also, verify with other devices.

Currently checking on iPad and mac as well.

If we use maxHeight, then it limit the size for that view for all device screens.

Reply:
Try with setting the layout priority of bottom bar to be higher. We set the layout priority of DerivationRecipe screen to be 1. That may be forcing the bottom bar to shrink